src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -251,9 +251,13 @@ Copyright (c) .NET Foundation. All rights reserved.
                                     Condition="'$(TieredCompilation)' != ''"
                                     Value="$(TieredCompilation)" />
 
-    <RuntimeHostConfigurationOption Include="System.Runtime.TieredCompilation.DisableTier0Jit"
-                                    Condition="'$(DisableTier0Jit)' != ''"
-                                    Value="$(DisableTier0Jit)" />
+    <RuntimeHostConfigurationOption Include="System.Runtime.TieredCompilation.StartupTierQuickJit"
+                                    Condition="'$(StartupTierQuickJit)' != ''"
+                                    Value="$(StartupTierQuickJit)" />
+
+    <RuntimeHostConfigurationOption Include="System.Runtime.QuickJitForLoops"
```

Comment:
I am not sure whether it make sense to have `QuickJitForLoops` exposed in SDK first class. I look at it as one of those config switches for experts only.